csrankings-c.csv

```diff
@@ -494,6 +494,7 @@ Chengqi Zhang,University of Technology Sydney,https://www.uts.edu.au/staff/cheng
 Chengqing Li,Hunan University,http://www.chengqingli.com,i4a0j8IAAAAJ
 Chengqing Zong,Chinese Academy of Sciences,http://www.nlpr.ia.ac.cn/cip/english/zong.htm,l8lvKOQAAAAJ
 Chengrong Wu,Fudan University,http://www.cs.fudan.edu.cn/en/?page_id=2253,NOSCHOLARPAGE
+Chenguang Wang,Washington University in St. Louis,https://cgraywang.github.io,hsZ2aj0AAAAJ

```

Comment:
Please add disambiguation suffix (0001) and I will merge.

Reply:
done, thanks!